src/components/standalone/openvpn_rw/RWServerDetails.vue

```diff
@@ -0,0 +1,110 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import { NeButton, NeDropdown } from '@nethserver/vue-tailwind-lib'
+import type { RWServer } from '@/views/standalone/vpn/OpenvpnRoadWarriorView.vue'
+
+const { t } = useI18n()
+
+defineProps<{
+  server: RWServer
+  connectedClients: number
+}>()
+
+const emit = defineEmits(['delete-server', 'edit-server'])
+</script>
+
+<template>
+  <div class="flex w-full flex-col gap-y-6 xl:flex-row xl:gap-x-6 xl:gap-y-0">
+    <div
+      class="flex grow flex-row items-center justify-between rounded-md border-l-4 border-indigo-400 bg-gray-100 p-2 dark:border-indigo-500 dark:bg-gray-800 sm:rounded-lg sm:shadow"
+    >
+      <div class="ml-4 flex flex-row items-center">
+        <font-awesome-icon
+          :icon="['fas', 'server']"
+          aria-hidden="true"
+          :class="`mr-5 h-4 w-4 rounded-full bg-gray-900 p-2 text-gray-300 dark:bg-gray-50 dark:text-gray-600`"
+        />
+        <p>{{ server.ns_description }}</p>
+      </div>
+      <div
+        class="ml-4 mr-10 grow border-l border-gray-800 py-3 pl-4 text-sm dark:border-gray-600 md:ml-8 md:pl-8"
+      >
+        <div class="flex max-w-xl flex-col justify-between gap-y-4 sm:flex-row sm:gap-x-4">
+          <div>
+            <p class="mb-2 font-semibold">{{ t('standalone.openvpn_rw.status') }}:</p>
+            <div :class="['flex', 'flex-row', 'items-center']">
+              <font-awesome-icon
+                :icon="['fas', true ? 'circle-check' : 'circle-xmark']"
```

Comment:
The icon is always green, even if the server is disabled.
```suggestion
                :icon="['fas', server.enabled == '1' ? 'circle-check' : 'circle-xmark']"
```